app/helpers/funnelcake_helper.rb

```diff
@@ -0,0 +1,8 @@
+module FunnelcakeHelper
+
+  def about_url()
+    config = YAML.load_file(File.expand_path("#{Rails.root}/config/funnelcake.yml", __FILE__))
```

Comment:
This should be done with `config_for` in an initializer or `config/application.rb`, like in [cob](https://github.com/tulibraries/tul_cob/blob/qa/config/application.rb#L24). Doing it here means we do IO every time the helper is called. Doing it in the initializer or application.rb means it only happens when the application boots up.

Reply:
Done.